odk1-src/aggregate-do.rst

```diff
@@ -1,117 +1,78 @@
+Create your Droplet
+-------------------
 
-  .. image:: https://github.com/opendatakit/aggregate/raw/master/cloud-config/digital-ocean/README_assets/DO_user_data_and_cloud_config.png
+.. tip::
+  If you have not already created a DigitalOcean account, use our referral link to do so: https://m.do.co/c/39937689124c.
 
-- Use the domain you want to use as the Droplet's name on the `Choose a hostname` section.
+  DigitalOcean will give you $100 of credit to spend during the first 60 days so that you can try things out. Once you have spent $25 with them, we’ll get $25 to put towards our hosting costs.
 
-  .. warning::
+1.  Log into Digital Ocean and create a new Droplet.
 
-    This data will be used by the Cloud-Config script to configure your server's domain name. You have to use the same domain to enable SSL in step 4.
+2.  Select the distribution for your new Droplet: Select the option :guilabel:`18.04 x64` from the Ubuntu box.
 
+    .. image:: /img/aggregate-do/distribution.*
+      :alt: Selecting the Droplet's distribution
 
-- Click on the `create` button
+3.  Choose a size fit for your intended usage. The :guilabel:`$5 Standard Droplet` should be enough for light ODK Aggregate use. If you find yourself needing more, Digital Ocean makes it easy to resize to a bigger Droplet.
 
-Although the creation of the Droplet itself takes just some seconds to complete, **the actual ODK Aggregate installation will take up to 10 minutes to complete**.
+4.  If you would like automatic weekly backups, enable them.
 
-In the mean time, you can continue with the next steps.
+5.  You will not need block storage.
 
-2 - Set up your domain
-~~~~~~~~~~~~~~~~~~~~~~
+6.  Choose a datacenter region physically close to where data collection
+    is going to happen.
 
-Once the Droplet is running, take note of its public IP address and set a *DNS A record* pointing to it:
+7.  Under :guilabel:`Select additional options`, check the :guilabel:`User data` checkbox. Copy and paste the contents of `this Cloud-Config script <https://raw.githubusercontent.com/opendatakit/aggregate/master/cloud-config/assets/cloud-config.yml>`_.
 
-- DigitalOcean `How to manage DNS records - A records <https://www.digitalocean.com/docs/networking/dns/how-to/manage-records/#a-records>`_
+    .. image:: /img/aggregate-do/user-data.*
+      :alt: Inserting Cloud-Config script under User Data section
 
-- Check your provider's instructions if your domain is not hosted or managed by DigitalOcean
 
-.. note::
+8.  In the :guilabel:`Choose a hostname section`, enter the domain name (e.g., your.domain). This hostname will be used by the Cloud-Config script to configure your server’s domain name. You must enter the same domain name to enable HTTPS.
 
-  - If you don't own a domain, services such as `FreeDNS <https://freedns.afraid.org>`_ offer creating subdomains under a range of domains for free.
+9.  You will not need to add public SSH keys (unless you know what that is and you want to).
```

Comment:
But then they won't be able to log into the machine and run the cerbot commands, update Aggregate...

Reply:
OK, I see now that DO sends you the root password